dandiapi/api/services/metadata/__init__.py

```diff
@@ -41,9 +41,6 @@ def validate_asset_metadata(*, asset: Asset) -> None:
         raise AssetHasBeenPublished()
 
     with transaction.atomic():
-        asset.status = Asset.Status.VALIDATING
-        asset.save()
```

Comment:
This seems to remove all use of the `VALIDATING` status then, correct? I'm actually in favor of this change, but if so, we should probably just remove that status from the model altogether.

Reply:
Will you allow me to do this in a follow on PR :pray:?

Reply:
Sure, no issue from me.